beluga/models.py

```diff
@@ -81,7 +82,7 @@ class Event(Base):
     start_time_local = sa.Column(sa.types.DateTime())
     end_time_local = sa.Column(sa.types.DateTime())
     timezone = sa.Column(sa.String(50))
-    location = sa.Column(sa.types.JSON)
+    location = sa.Column(Geography('POINT', srid=4326))
```

Comment:
I'm inclined to not include `srid` unless there is a reason to do so and trust PostGIS's defaults

Reply:
Also I think this should be `geometry_type="POINT"`, not just a string argument, per [the docs](https://geoalchemy-2.readthedocs.io/en/latest/types.html#geoalchemy2.types.Geography)